.storybook/webpack.config.js

```diff
@@ -5,13 +5,25 @@
 // IMPORTANT
 // When you add this file, we won't add the default configurations which is similar
 // to "React Create App". This only has babel loader to load JavaScript.
+const Webpack = require('webpack');
 
-module.exports = function({ config }) {
+module.exports = function({ config, mode }) {
   config.module.rules.push({
     test: /\.stories.jsx?$/,
     loaders: [require.resolve('@storybook/addon-storysource/loader')],
     enforce: 'pre',
   });
 
+  const ENV_DEV = 'development';
+  const ENV_PROD = 'production';
```

Comment:
Looks like `ENV_PROD` is not used.

Reply:
I believe it's only being used in `Logger`

Reply:
How? It's just a local constant, and not used anywhere in the file.